game/armorGroupLocation.py

```diff
@@ -0,0 +1,7 @@
+from game.point_with_heading import PointWithHeading
```

Comment:
Wrong naming convention for the file. `armorgrouplocation.py`

Reply:
changed